scipy/spatial/transform/_rotation.pyx

```diff
@@ -433,6 +501,48 @@ cdef double[:, :] _elementary_quat_compose(
                 result)
     return result
 
+def _format_angles(angles, degrees, num_axes):
```

Comment:
Check strings alignment in this function.

Reply:
Just fixed!